### PR TITLE
[NETBEANS-3377] - cleanup unchecked conversions

### PR DESCRIPTION
**Pull request by BradWalker**
There are a lot of places in the code where unchecked conversions are taking place..

   [repeat] /home/bwalker/src/netbeans/profiler/profiler.heapwalker/src/org/netbeans/modules/profiler/heapwalk/OQLController.java:259: warning: [unchecked] unchecked conversion
   [repeat]             Set<Map.Entry> entries = ((Map)o).entrySet();
   [repeat]                                                       ^
   [repeat]   required: Set<Entry>
   [repeat]   found:    Set

Cleanup these warning messages..

### CODE REVIEW

**Comment by matthiasblaesing** [comment]
@BradWalker could you please rebase this changeset on current master? I suspect there were changes in the newer PR, that interfere here.

File: ide/spi.debugger.ui/src/org/netbeans/modules/debugger/ui/DebuggerManagerListener.java [hunk 1]
```diff
@@ -673,7 +673,11 @@ private static void doCloseDebuggerUI() {
             final Set<Component> initiallyOpenedMinimized;
             if (componentsInitiallyOpened.isEmpty()) {
                 initiallyOpened = Collections.emptyList();
+<<<<<<< HEAD
                 initiallyOpenedMinimized = Collections.EMPTY_SET;
+=======
+                initiallyOpenedMinimized = Collections.emptySet();
+>>>>>>> [NETBEANS-3377] - cleanup unchecked conversions
```
Comment (matthiasblaesing) [inline ide/spi.debugger.ui/src/org/netbeans/modules/debugger/ui/DebuggerManagerListener.java]: This is a merge conflict, that was not resolved - please check again.

Comment (BradWalker) [reply ide/spi.debugger.ui/src/org/netbeans/modules/debugger/ui/DebuggerManagerListener.java]: UGG!! Must have missed something. Sorry about that.. Should be fixed down..

**Comment by matthiasblaesing** [comment]
Merged via https://github.com/apache/netbeans/commit/e1ef6a5692716db1efda223cec99d8c6981cf131. Thank you.

**Pull request closed without merging**